conan/tools/meson/toolchain.py

```diff
@@ -450,6 +498,15 @@ def _sanitize_format(v):
             "is_apple_system": self._is_apple_system
         }
 
+    @property
+    def _filename(self):
+        if self._is_cross_building and self._native:
```

Comment:
This filename before was conditioned to the user defining ``self.cross_build``, but now still it only uses the ``cross_building`` call?